image-injection/src/test/java/pages/ios/SwagLabsPage.java

```diff
@@ -0,0 +1,75 @@
+package pages.ios;
+
+import helpers.utils;
+import io.appium.java_client.AppiumDriver;
+import io.appium.java_client.ios.IOSDriver;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.util.Set;
+
+public class SwagLabsPage {
+
+    By usernameID = By.id("test-Username");
+    By passwordID = By.id("test-Password");
+    By submitButton = By.id("test-LOGIN");
+    By ProductTitle = By.xpath("//XCUIElementTypeStaticText[@name=\"PRODUCTS\"]\n");
```

Comment:
Noooooo!!!

Please don't use XPATH for iOS, try `-ios predicate string` or `-ios class chain` to speed up the tests

Reply:
Thanks @wswebcreation . I changed the iOS XPath locators. Good feedback :-)

Reply:
You're welcome